src/legacy/server/index_patterns/service/lib/field_capabilities/field_caps_response.ts

```diff
@@ -75,26 +88,31 @@ import { shouldReadFieldFromDocValues } from './should_read_field_from_doc_value
  *    }
  *
  *  @param {FieldCapsResponse} fieldCapsResponse
- *  @return {Promise<Array<FieldInfo>>}
```

Comment:
By the looks of things this was a typo right? Doesn't appear that this ever returned a promise to begin with.

Reply:
That's what I was seeing